test/cli/test_log_runtime_archive_main.rb

```diff
@@ -128,6 +129,118 @@ def call_archive(root_path, archive_path, low_limit, freed_limit)
                 end
             end
 
+            describe "#transfer_server" do
+                before do
+                    @target_dir = make_tmppath
+                    @server_params = server_params
+                end
+
+                it "creates an FTP server" do
+                    flexmock(Runtime::Server::SpawnServer)
+                        .new_instances
+                        .should_receive(:initialize).with(
+                            @target_dir, *@server_params.values
+                        ).once
```

Comment:
This doesnt test anything, I am not even sure how this passes.IMO the best way to test that the server was created is to send a noop command to it after starting it normally through the CLI.

Reply:
Opening the connection with the server is fine too